docs/zh_cn/user_guides/visualization_feature_map.md

```diff
@@ -0,0 +1,201 @@
+        gt_mask = PixelData(**gt_mask)
+        data_sample = SegDataSample()
+        data_sample.gt_sem_seg = gt_mask
+
+        seg_visualizer.add_datasample(
+            name='gt_mask',
+            image=image,
+            data_sample=data_sample,
+            draw_gt=True,
+            draw_pred=False,
+            wait_time=0,
+            out_file=None,
+            show=False)
+
+    seg_visualizer.add_image('image', image)
+
+
+def main():
+    parser = ArgumentParser(
+        description='Draw the Feature Map During Inference')
+    parser.add_argument('img', help='Image file')
+    parser.add_argument('config', help='Config file')
+    parser.add_argument('checkpoint', help='Checkpoint file')
+    parser.add_argument('--gt_mask', default=None, help='Path of gt mask file')
+    parser.add_argument('--out-file', default=None, help='Path to output file')
+    parser.add_argument(
+        '--device', default='cuda:0', help='Device used for inference')
+    parser.add_argument(
+        '--opacity',
+        type=float,
+        default=0.5,
+        help='Opacity of painted segmentation map. In (0, 1] range.')
+    parser.add_argument(
+        '--title', default='result', help='The image identifier.')
+    args = parser.parse_args()
+
+    register_all_modules()
+
+    # build the model from a config file and a checkpoint file
+    model = init_model(args.config, args.checkpoint, device=args.device)
+    if args.device == 'cpu':
+        model = revert_sync_batchnorm(model)
+
+    # show all named module in the model and use it in source list below
+    for name, module in model.named_modules():
+        print(name)
+
+    source = [
+        'decode_head.fusion.stages.0.query_project.activate',
+        'decode_head.context.stages.0.key_project.activate',
+        'decode_head.context.bottleneck.activate'
+    ]
+    source = dict.fromkeys(source)
+
+    count = 0
+    recorder = Recorder()
+    # registry the forward hook
+    for name, module in model.named_modules():
+        if name in source:
+            count += 1
+            module.register_forward_hook(recorder.record_data_hook)
+            if count == len(source):
+                break
+
+    with recorder:
+        # test a single image, and record feature map to data_buffer
+        result = inference_model(model, args.img)
+
+    visualize(args, model, recorder, result)
+
+
+if __name__ == '__main__':
+    main()
+
+```
+
+将上述代码保存为visualization_feature_map.py，在终端执行如下代码
+
+```shell
+python feature_map_visual.py ${图像} ${配置文件} ${检查点文件} [可选参数]
+```
+
+样例
+
+```shell
+python feature_map_visual.py \
+aachen_000000_000019_leftImg8bit.png \
+configs/ann/ann_r50-d8_4xb2-40k_cityscapes-512x1024.py \
+ann_r50-d8_512x1024_40k_cityscapes_20200605_095211-049fc292.pth \
+--gt_mask aachen_000000_000019_gtFine_labelTrainIds.png
+```
+
+可视化后的图像结果和它的对应的 feature map图像会出现在wandb账户中
+
+<div align=center>
+<img src="../../../demo/wandb_vis_feature_map.png">
```

Comment:
This pic is too big to be included in the repo, please use the picture-bed of github.

Reply:
<div align=center>
<img src="https://user-images.githubusercontent.com/81373517/216981376-20f13605-ecd4-4dd0-91f7-08ab0753cd02.png"/>
</div>

Reply:
https://user-images.githubusercontent.com/81373517/216981376-20f13605-ecd4-4dd0-91f7-08ab0753cd02.png

Reply:
Can use this one?

Reply:
Yes